tests/phpunit/Unit/ListItemTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\ListItem;
+use Tests\TestCase;
+use App\Enums\ListType;
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+class ListItemTest extends TestCase
+{
+    public function testThatItCanCreateAListItemFromACorrectUrlParameter()
+    {
+        $listItem = ListItem::createFromUrlParameter('999-base:id');
+        $this->assertInstanceOf(ListItem::class, $listItem);
+        $this->assertCount(4, (array)$listItem);
```

Comment:
I'd `assertEquals()` and check if the array matches the expected. This implicitly checks the count and also tests that it actually returns the expected and not just an array of four empty strings.